cli/foundation-models/system/finetune/text-generation/text-generation.sh

```diff
@@ -0,0 +1,191 @@
+set -x
+# the commands in this file map to steps in this notebook: https://aka.ms/azureml-ft-sdk-text-generation
+# the data files are available in the same folder as the above notebook
+
+# script inputs
+subscription_id="<SUBSCRIPTION_ID>"
+resource_group_name="<RESOURCE_GROUP>"
+workspace_name="WORKSPACE_NAME>"
+registry_name="azureml"
+model_registry_name="azureml-meta"
+
+compute_cluster="gpu-cluster-big"
+# if above compute cluster does not exist, create it with the following vm size
+compute_sku="Standard_NC24rs_v3"
+# This is the number of GPUs in a single node of the selected 'vm_size' compute. 
+# Setting this to less than the number of GPUs will result in underutilized GPUs, taking longer to train.
+# Setting this to more than the number of GPUs will result in an error.
+# CLI command to fetch the number of GPUs
+# az ml compute list-sizes -g <resource_group_name> -w <workspace_name> --subscription <subscription_id> --query "[?name=='<compute_sku>'].{Name:name, Gpus:gpus}" --output table
+gpus_per_node=8
```

Comment:
`Standard_NC24rs_v3` has 4 gpus per node. Isn't it?

Reply:
Changed to ND40 as that is the finetuning sku we are mentioning for text-gen